types/base/api.d.ts

```diff
@@ -1,15 +1,19 @@
-type Part = {
+import { Order } from "../api";
+
+export type Part = {
     name: string;
     material: string;
     color: string;
+    hidden?: boolean;
+    optional?: boolean;
 }
 
-type InitialsExtra = Record<string, {
+export  type InitialsExtra = Record<string, {
```

Comment:
Doesn't prettier fix this?

```suggestion
export type InitialsExtra = Record<string, {
```

Reply:
Apparently, `prettier` was not analyzing types, I fixed `package.json` script 👍

Reply:
@joao-conde check if that fix needs to be ported to other repositories